package.json

```diff
@@ -91,6 +91,7 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.12.1",
+    "@mdi/font": "^5.9.55",
```

Comment:
@ahochsteger Why this? It makes the application much heavier, couldn't you use the https://material.io/resources/icons/?search=content&icon=content_copy&style=baseline ?

Reply:
You are right, that it is not required for the copy icon - I just had troubles in the past to find good icons (e.g. for formatting columns) and found out how to use better icons while working at it.
I'll remove it, since it has nothing to do with this PR.

Reply:
done.